Project 3 Lol

```diff
@@ -0,0 +1,350 @@
+...
+- Computer1 instance_of object
+- Computer2 instance_of object
+- Computer3 instance_of object
+...
+- Wifi_network instance_of object aggregate (project 3 document)
+- Wifi_network connects Computer1
+- Computer1 connected_to Computer2
+- Computer1 connected_to Computer3
+...
+- Turtle_Serialization generically_depends_on Computer1
+- XML_Serialization generically_depends_on Computer1
+- Turtle_Serialization generically_depends_on Computer2
+- XML_Serialization generically_depends_on Computer2
+- Turtle_Serialization generically_depends_on Computer3
+- XML_Serialization generically_depends_on Computer3
+
+6. Using Protege, place these in the BFO hierarchy where you think they fit best:
+```
+  (a) Bach's Well-Tempered Clavier
+  - Process
+  - the sheet music, along with any physical representation of the song (including recordings), are generically dependent continuants.
+  
+  (b) Chair of the UB Philosophy Department
+  - Role
+  -- The many chairs inside the UB Philosophy Department fiat object part of UB's campus are objects.
+  
+  (c) SARS-CoV-2
+  - object
+  
+  (d) Mexico City
+  - object aggregate made up of objects like inhabitants, buildings, streets, lights, smells, and et cetera.
+  - What I am confused about here is whether or not an object aggregate must be aggregated of all the same object, as this is the case with the examples listed in chapter 5.
+  
+  (e) The trunk of a minivan
+  - fiat object part
+  - page 113 lists "the trunk of your car" as an example of a site.
+  
+  (f) Occupation
+  - Depends, the position, as in the formal title, is a role. 
+  - The conditions and expectations set around a position are generically dependent continuants about the role.
+  - the actual work done, one's occupation of their time, is a process.
+  
+  (g) Ocean
+  - fiat object part
+  
+  (h) Lake
+  - fiat object part
+...
+- My answers for (g) and (h) are informed by the attached article: https://journals.sagepub.com/doi/pdf/10.1177/0263775817700395.
+- I've heard Giacomo and Ali categorize both oceans and lakes as sites according to BFO. I support this categorization.
+  
+```
+7. True or False; explain your answers:
+```
+  (a) An instance of Material Entity can have an instance of Immaterial Entity as part.
+  - True: I think holes and cavities are immaterial entities.
+  - My answer here is informed by chapter 5 of the BFO textbook as well as the attached slides: https://buffalo.app.box.com/s/6wmxnd9vvuhd9ubrldeptaerydqzkbct
+  
+  (b) An instance of Immaterial Entity can have an instance of Material Entity as part.
+  - True: an imaginary friend is in a symmetric parthood relationship with the mind of the child who imagines it.
+  - My take is informed by page 112 of the BFO textbook.
+ ...
+  - Giacomo pointed out how the classes of material and immaterial entities are disjoint, according to BFO.
+  - If this is the case, my answers to both (a) and (b) above are false... am stuck now
```

Comment:
They are disjoint but this doesn't mean that they can't have each other as parts. So a is definitely true: I can have a point as part. But I believe b is false. Immaterial entities in BFO are things like points, lines, etc. rather than tings like abstract entities or imaginary friends. These would rather be generically dependent continuants. In the definition of immaterial entities I believe it is explicitly said that they cannot have material entities as parts. Because imagine me being a part of something like a line. It doesn't make much sense. I can be located in it, but not be a part of it.